core/src/main/java/com/datastax/oss/driver/api/core/cql/StatementBuilder.java

```diff
@@ -123,6 +124,11 @@ public SelfT setRoutingKey(@Nullable ByteBuffer routingKey) {
     return self;
   }
 
+  @NonNull
```

Comment:
Other methods in this class have this kind of javadocs:

```
  /** @see Statement#setRoutingKey(ByteBuffer) */
```

So I think we should do the same here.

Reply:
Agreed; I saw those and simply forgot to make sure they were here.  I'll add those in for sure.